benchmark/error/node-error.js

```diff
@@ -3,19 +3,24 @@
 const common = require('../common');
 
 const bench = common.createBenchmark(main, {
-  n: [1e7],
+  n: [1e5],
+  type: ['node', 'regular'],
 }, {
   flags: ['--expose-internals'],
 });
 
-function main({ n }) {
+function main({ n, type }) {
   const {
     codes: {
       ERR_INVALID_STATE,
     },
   } = require('internal/errors');
+
+  const clazz = type === 'node' ? ERR_INVALID_STATE.TypeError : TypeError;
```

Comment:
nit: constructors (and aliases to them) are usually PascalCase

```suggestion
  const Clazz = type === 'node' ? ERR_INVALID_STATE.TypeError : TypeError;
```

Reply:
Done